src/ivory-px4-hw/SMACCMPilot/Hardware/MPU6000/SPI.hs

```diff
@@ -180,19 +235,20 @@ mpu6000SensorManager (BackpressureTransmit req_chan res_chan) init_chan gyro_cha
         comment $ "sample rate: " ++ show (1000 / fromInteger samplePeriodMS :: Double) ++ "Hz"
         _ <- rpc $ writeRegReq dev SampleRateDivider $ fromInteger divisor
 
-        comment "Set accelerometer scale to +/- 16g"
-        _ <- rpc (writeRegReq dev AccelConfig 0x18)
+--        comment "Set accelerometer scale to +/- 16g"
+--        _ <- rpc (writeRegReq dev AccelConfig 0x18)
+        comment "Set accelerometer scale to +/- 8"
+        _ <- rpc (writeRegReq dev AccelConfig 0x10)
```

Comment:
Do we have the right scale factor for +/-8g? It looks like we're now dividing the raw ADC by twice as much as before, shouldn't we divide by half as much given the smaller scale?